utils/outputwriter/outputcontent.go

```diff
@@ -118,288 +157,719 @@ func footer(writer OutputWriter) string {
 	return fmt.Sprintf("%s\n%s", SectionDivider(), writer.MarkInCenter(CommentGeneratedByFrogbot))
 }
 
-func VulnerabilitiesContent(vulnerabilities []formats.VulnerabilityOrViolationRow, writer OutputWriter) (content []string) {
-	if len(vulnerabilities) == 0 {
-		return []string{}
+func GetFrogbotErrorCommentContent(contentForComments []string, err error, writer OutputWriter) (comments []string) {
+	// First decorate with error suffix, then wrap content with the base decorator
+	return ConvertContentToComments(contentForComments, writer, getFrogbotErrorSuffixDecorator(writer, err), GetFrogbotCommentBaseDecorator(writer))
+}
+
+// Adding markdown prefix to identify Frogbot comment and a footer with the link to the documentation
+func getFrogbotErrorSuffixDecorator(writer OutputWriter, err error) CommentDecorator {
+	return func(_ int, content string) string {
+		var comment strings.Builder
+		WriteNewLine(&comment)
+		// Error
+		WriteContent(&comment, writer.MarkAsTitle("Error:", 4), err.Error())
+		WriteNewLine(&comment)
+		// Action steps
+		WriteContent(&comment,
+			writer.MarkAsTitle("Next Steps:", 4),
+			"1. Please try to rerun the scan.",
+			fmt.Sprintf("2. If the issue persists, consider checking the %s for troubleshooting tips.", MarkAsLink("Frogbot documentation", FrogbotDocumentationUrl)),
+			fmt.Sprintf("3. If you still need assistance, feel free to reach out to %s.", MarkAsLink("JFrog Support", JfrogSupportUrl)),
+		)
+		WriteNewLine(&comment)
+		WriteContent(&comment, "Thank you for your understanding!")
+
+		return content + writer.MarkAsDetails(jobErrorTitle, 3, comment.String())
 	}
-	content = append(content, writer.MarkAsTitle(vulnerableDependenciesTitle, 2))
-	content = append(content, vulnerabilitiesSummaryContent(vulnerabilities, writer))
-	content = append(content, vulnerabilityDetailsContent(vulnerabilities, writer)...)
-	return
 }
 
-func vulnerabilitiesSummaryContent(vulnerabilities []formats.VulnerabilityOrViolationRow, writer OutputWriter) string {
+// Summary content
+
+func ScanSummaryContent(issues issues.ScansIssuesCollection, violationContext string, includeSecrets bool, writer OutputWriter) string {
+	if !issues.IssuesExists(includeSecrets) && !issues.HasErrors() {
+		return ""
+	}
 	var contentBuilder strings.Builder
-	WriteContent(&contentBuilder,
-		writer.MarkAsTitle("✍️ Summary", 3),
-		writer.MarkInCenter(getVulnerabilitiesSummaryTable(vulnerabilities, writer)),
+	totalIssues := issues.GetTotalVulnerabilities(includeSecrets)
+	violations := false
+	if violationContext != "" && violationContext != NoViolations {
+		totalIssues = issues.GetTotalViolations(includeSecrets)
+		violations = true
+	}
+	title := MarkAsBullet(fmt.Sprintf("Frogbot scanned for %s and found %d issues", violationContext, totalIssues))
+	if issues.HasErrors() {
+		title = MarkAsBullet(fmt.Sprintf("Frogbot attempted to scan for %s but encountered an error.", violationContext))
```

Comment:
Do we want to change the title upon encountering any error in any scanner? what about cases that only one scanner failed but the rest were successful?
It looks different than the picture you added to the PR, where we have a failure in CA but we get the title like we had no errors